CONTRIBUTING.md

```diff
@@ -0,0 +1,16 @@
+# Contributing
+
+Pull requests are welcomed. You must
+
+- Sign the Elastic [Contributor License Agreement](https://www.elastic.co/contributor-agreement).
+- Include a [changelog][changelog_docs] entry at `.changelog/{pr-number}.txt` with your pull request.
+- Include tests that demonstrate the change is working.
+
+[changelog_docs]: https://github.com/GoogleCloudPlatform/magic-modules/blob/2834761fec3acbf35cacbffe100530f82eada650/.ci/RELEASE_NOTES_GUIDE.md#expected-format
```

Comment:
I don't see this line at all on the rendered github view: https://github.com/andrewkroh/stream/blob/release-workflow/CONTRIBUTING.md

But maybe there could be a template or examples of the format in this repo. I don't find the linked page very clear. Should there be a PR description before the release note? From the changelog in this PR, I guess  not, but the doc makes it seem like it should.

Reply:
I have added more explanation and examples in the CONTRIBUTING.md. After a few changes are made there will be many examples to follow in the .changelog/ dir.